packages/components/src/color-picker/inputs.js

```diff
@@ -103,12 +103,12 @@ export class Inputs extends Component {
 		} else if ( this.state.view === 'rgb' ) {
 			this.setState( { view: 'hsl' } );
 
-			speak( __( 'Hue/saturation/lightness mode active' ) );
+			speak( __( 'HSL mode active' ) );
 		} else if ( this.state.view === 'hsl' ) {
 			if ( this.props.hsl.a === 1 ) {
 				this.setState( { view: 'hex' } );
 
-				speak( __( 'Hex color mode active' ) );
+				speak( __( 'HEX color mode active' ) );
```

Comment:
Hex isn't an acronym like HSL or RGB and should not be capitalized.

Reply:
Fixed.